package.json

```diff
@@ -26,6 +27,7 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-scripts": "4.0.2",
+    "react-use-measure": "^2.0.3",
```

Comment:
did not see where this was used in code

Reply:
I was experimenting with this and forgot to remove it, good catch!